nuxeo-runtime/nuxeo-connect-standalone/src/main/java/org/nuxeo/launcher/connect/ConnectBroker.java

```diff
@@ -1047,6 +1117,10 @@ public boolean executePending(File commandsFile, boolean doExecute, boolean useR
                 if (errorValue != 0) {
                     log.error("Error processing pending package/command: " + line);
                 }
+                if (doExecute && !useResolver && isRestartRequired()) {
+                    remainingCmds.forEach(this::persistPendingCommand);
```

Comment:
vs `persistPendingCommand(String, Collection<String>)` ?
I guess that the answer will point to the issue/mistake in the method signatures